src/Serialization/GAP.jl

```diff
@@ -233,33 +233,31 @@ install_GAP_serialization(:IsPcGroup,
                   for x in fullpcgs]
         save_object(s, relord, :relord)
         # power relators
-        reli = []
-        rels = []
+        rels = Tuple{Int, Vector{Int}}[]
         for i in 1:length(relord)
           ne = fullpcgs[i]^relord[i]
           if ! GAP.Globals.IsOne(ne)
-            push!(reli, i)
-            push!(rels, _free_group_extrep_from_exponents(
-              Vector{Int}(GAP.Globals.ExponentsOfPcElement(fullpcgs, ne))))
+            push!(rels, (i, _free_group_extrep_from_exponents(
+              Vector{Int}(GAP.Globals.ExponentsOfPcElement(fullpcgs, ne)))))
           end
         end
-        save_object(s, reli, :power_reli)
-        save_object(s, rels, :power_rels)
+        if length(rels) != 0
+          save_typed_object(s, rels, :power_rels)
```

Comment:
```suggestion
          save_data_array(s, :power_rels) do
            for rel in rels do
              save_object(s, rel)
            end
          end
```

Reply:
```suggestion
          save_object(s, rels, :power_rels)
```
this should actually work better